src/login/login.component.tsx

```diff
@@ -28,75 +43,62 @@ export default function Login(props: LoginProps) {
     }
   }, [showPassword, passwordInputRef.current, usernameInputRef.current]);
 
-  const input = css`
-    width: 100%;
-    padding: 10px 5px;
-    box-sizing: border-box;
-    font-size: 14px;
-    margin: 10px 0px 10px 0px;
-    border-radius: 4px;
-  `;
-
   return (
     <div
+      className="canvas"
       css={css`
         display: flex;
         justify-content: center;
         align-items: center;
         width: 100vw;
         height: 100vh;
-        background: #fafbfc;
       `}
     >
-      <div
-        className="omrs-card omrs-padding-16"
-        css={css`
-          width: 300px;
-        `}
-      >
-        <svg role="img">
-          <use xlinkHref="#omrs-logo-full-color" />
-        </svg>
-        <form
-          onSubmit={handleSubmit}
+      <div className="omrs-card omrs-padding-16">
+        <div
           css={css`
-            margin: 10px;
+            text-align: center;
           `}
         >
-          <div>
-            <label>
-              <input
-                css={css`
-                  ${input}
-                `}
-                type="text"
-                name="username"
-                value={username}
-                onChange={evt => setUsername(evt.target.value)}
-                placeholder="Username"
-                ref={usernameInputRef}
-                autoFocus
-              />
-            </label>
+          <svg role="img">
+            <use xlinkHref="#omrs-logo-full-color"></use>
+          </svg>
+        </div>
+        <form
+          onSubmit={handleSubmit}
+          className="omrs-margin-top-32"
+          ref={formRef}
+        >
+          <div className="omrs-input-group">
+            <input
+              id="username"
+              className={`omrs-input-outlined ${
+                authenticated === false ? "omrs-input-danger" : ""
```

Comment:
Not necessary to change this, but I actually like using the kremling library's syntax for this kind of thing:

```jsx
<input className={always("omrs-input-outlined").maybe("omrs-input-danger", authenticated === false)} />`
```

See https://kremling.js.org/api/always-maybe-and-toggle.html. I'm obviously biased towards it, being the author of it though 😄

Reply:
looks super cool 😎 I'm open to trying it out right away